connectivity/drivers/cellular/QUECTEL/BC95/QUECTEL_BC95_CellularStack.cpp

```diff
@@ -225,8 +225,8 @@ nsapi_size_or_error_t QUECTEL_BC95_CellularStack::socket_sendto_impl(CellularSoc
 
     // check for network congestion
     device_err_t err = _at.get_last_device_error();
-    if (err.errType == DeviceErrorTypeErrorCME &&
-            (err.errCode == AT_UART_BUFFER_ERROR || err.errCode == AT_BACK_OFF_TIMER) || err.errCode == AT_UPLINK_BUSY) {
+    if ((err.errType == DeviceErrorTypeErrorCME &&
+            (err.errCode == AT_UART_BUFFER_ERROR || err.errCode == AT_BACK_OFF_TIMER)) || err.errCode == AT_UPLINK_BUSY) {
```

Comment:
The indentation makes it difficult to immediately see what is going on here. Does the following improve things a bit?
```suggestion
    if (
        (
            err.errType == DeviceErrorTypeErrorCME
            && (err.errCode == AT_UART_BUFFER_ERROR || err.errCode == AT_BACK_OFF_TIMER)
        )
        || err.errCode == AT_UPLINK_BUSY
    ) {
```